extra/release_tool.py

```diff
@@ -263,6 +272,9 @@ def is_independent_component(self):
 BACKEND_SERVICES = (
     BACKEND_SERVICES_OPEN | BACKEND_SERVICES_ENT | BACKEND_SERVICES_OPEN_ENT
 )
+CLIENT_SERVICES_ENT = {
```

Comment:
Is this strictly necessary? :cry: 

Update note in [point 3 here](https://github.com/mendersoftware/mender-qa/blob/master/Documentation/adding-new-repository.md#integration) so that it is followed for all repos (not only backend repos)

Reply:
Thanks, I'll do that!

For whether it's necessary, see [this comment](https://github.com/mendersoftware/integration/pull/1836#issuecomment-1005715282). I welcome other approaches, this part is tricky because it is invoked so many times as a sub command.